frontend/src/routes/Governance/policies/EditPolicy.tsx

```diff
@@ -30,6 +30,7 @@ export function WizardSyncEditor() {
       resources={resources}
       schema={schema}
       filters={['*.metadata.managedFields']}
+      immutables={['PlacementBinding.0.subjects.0.*']}
```

Comment:
This makes it look way easier than I was expecting! I wonder if the entire PlacementBinding should be immutable... It is helpful to see the binding on the "Edit Policy" page - just to have a full view of what is necessary for the Policy - but I think we might want to discourage changing any of its fields because it's really easy to mess something up.

Reply:
sounds good :)